examples/relation.rs

```diff
@@ -0,0 +1,54 @@
+extern crate osmpbfreader;
+use std::collections::HashSet;
+
+fn wanted(obj: &osmpbfreader::OsmObj) -> bool{
+    match *obj {
+        osmpbfreader::OsmObj::Relation(ref rel) => {
+            if rel.id == 7444 {
+                return true;
+            }else{
+                return false;
+            }
+        }
+        _ => {return false;}
+    }
+}
+
+fn main() {
+    let filename = std::env::args_os().nth(1).unwrap();
+    let path = std::path::Path::new(&filename);
+    let r = std::fs::File::open(&path).unwrap();
+    let mut pbf = osmpbfreader::OsmPbfReader::new(r);
+    let mut finished = false;
+    let mut dependancies = HashSet::new();
+    while !finished{
+        finished = true;
+        for _obj in pbf.iter() {
```

Comment:
beginning a variable with a `_` means you don't use it. So you should rename `_obj` by `obj`